src/mixins/Mime.js

```diff
@@ -71,6 +79,15 @@ export default {
 		}
 	},
 
+	computed: {
+		name() {
+			return this.fileName.split('.')[0]
+		},
+		ext() {
+			return this.fileName.split('.')[1]
```

Comment:
does :boom: for files without an extension

Reply:
No, it will be undefined, which is good since the extension is, well, not defined :)